app-demo/src/main/java/co/sodalabs/demo/DemoActivity.kt

```diff
@@ -21,3 +53,34 @@ class DemoActivity : AppCompatActivity() {
         mDisposablesOnCreate.clear()
     }
 }
+
+class DemoAdapter(private val items: List<DemoViewModel>) : RecyclerView.Adapter<DemoAdapter.ViewHolder>() {
+
+    var clickListener: ((view: View, item: DemoViewModel) -> Unit)? = null
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) =
+        ViewHolder(LayoutInflater.from(parent.context).inflate(R.layout.demo_recycler_view_item, parent, false))
+
+    override fun getItemCount(): Int = items.size
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        holder.bind(items[position])
+        holder.setOnClickListener(clickListener)
+
```

Comment:
MINOR: Remove this empty line.

Reply:
I'd be very happy to have it. Is it possible to integrate the lint quickly?

Reply:
Yup! Ktlint is really easy to setup. Just run the command from here:
https://github.com/shyiko/ktlint#installation and it'll take care of rest.

Reply:
Let's leave the ktlint integrate another ticket.